lib/file.cpp

```diff
@@ -26,6 +27,12 @@ std::string file$bytes(const std::string &filename)
     return r;
 }
 
+bool file$exists(const std::string &filename)
+{
+    struct stat buffer;
+    return stat(filename.c_str(), &buffer) == 0; 
```

Comment:
For an existence check, use the access() function instead of stat(). We don't need all the extra crap returned by stat().